### PR TITLE
Added template

### PR DESCRIPTION
**Pull request by ninaandal**
Fixed the `starter.js` to include fields for the new v3 template

### CODE REVIEW

**Review by RitaDias — CHANGES_REQUESTED** [review]
image

To double check, we decided to go with cssFrameworks being mandatory? @mariuslundgard

image

Maybe a nitpick but should we update the validation messages at least for those fields since they are bit more tech-y than they probably need to?

**Comment by RitaDias** [comment]
Also makes me wonder what do we want to do with the previews? I assume that we didn't end up doing anything with them but I'm guess that they will be wrong for the v3 templates? @mariuslundgard do you have any thoughts?

**Comment by RitaDias** [comment]
image

If I switch from v3 (after having added a title and seen the validation errors) -> v2 templates the validation errors from v3 are still appearing on the validation tab

**Comment by ninaandal** [comment]
> image
> 
> To double check, we decided to go with cssFrameworks being mandatory? @mariuslundgard
> 
> image
> 
> Maybe a nitpick but should we update the validation messages at least for those fields since they are bit more tech-y than they probably need to?

No, this is not nitpick - thanks for pointing this out! I have fixed this 👍 

About the CSS Framework - yes, but we will include an "Other" option if it is not applicable

**Comment by ninaandal** [comment]
> Also makes me wonder what do we want to do with the previews? I assume that we didn't end up doing anything with them but I'm guess that they will be wrong for the v3 templates? @mariuslundgard do you have any thoughts?

Won't we be able to show the sanity page you are making as an `iframe`?

**Comment by RitaDias** [comment]
> 
> Won't we be able to show the sanity page you are making as an `iframe`?

That is the plan yes! I might have missed it but I don't think any code about the preview was charged in this PR right? Which I'm assuming that then it won't work out of the box for v3 or maybe I missed something?

**Comment by RitaDias** [comment]
image

still getting a v3 error when I go from v3 -> v2 (relative address in the community website field)
Actually, the field appearing for v2 overall is an error, isn't it? I think that field shouldn't be appearing in v2 🤔 

image

Also, it looks like there are some file conflicts 👇

File: schemas/documents/contributions/starter.js [hunk 1]
```diff
@@ -10,130 +9,53 @@ import {
...
+      type: 'number',
+      description: 'What Sanity Studio version was this starter was built for.',
+      initialValue: 3,
+      options: {
+        layout: 'radio',
+        direction: 'horizontal',
+        list: [
+          {value: 3, title: 'Studio v3'},
+          {value: 2, title: 'Studio v2'},
+          {value: -1, title: 'n/a'},
```
Comment (RitaDias) [inline schemas/documents/contributions/starter.js:31]: While looking at this I was wondering if it wouldn't make sense to also add the deprecated to the field directly. What do you think @mariuslundgard 
```suggestion
          {value: 2, title: 'Studio v2 (deprecated)'},
          {value: -1, title: 'n/a (deprecated)'},
```

**Comment by ninaandal** [comment]
> image
> 
> still getting a v3 error when I go from v3 -> v2 (relative address in the community website field) Actually, the field appearing for v2 overall is an error, isn't it? I think that field shouldn't be appearing in v2 🤔
> 
> image
> 
> Also, it looks like there are some file conflicts 👇

But this is the `slug` field, which I thought was mandatory for both v2 and v3?

**Comment by ninaandal** [comment]
> > Won't we be able to show the sanity page you are making as an `iframe`?
> 
> That is the plan yes! I might have missed it but I don't think any code about the preview was charged in this PR right? Which I'm assuming that then it won't work out of the box for v3 or maybe I missed something?

No, I haven't done anything with the previews yet, as you weren't done with the page yet! Mind if we do it in a separate PR after this is merged?

**Comment by RitaDias** [comment]
> > > Won't we be able to show the sanity page you are making as an `iframe`?
> > 
> > 
> > That is the plan yes! I might have missed it but I don't think any code about the preview was charged in this PR right? Which I'm assuming that then it won't work out of the box for v3 or maybe I missed something?
> 
> No, I haven't done anything with the previews yet, as you weren't done with the page yet! Mind if we do it in a separate PR after this is merged?

Just so you all know, we've spoken and a new task was created to update the preview / make sure the preview works for the new v3

**Review by RitaDias — APPROVED** [review]
✅ let's go!

File: schemas/documents/contributions/starter.js [hunk 2]
```diff
@@ -295,6 +294,7 @@ export default {
         title: 'Sanity tools this starter relies on',
         description:
           'Browse for plugins, asset sources, SDKs and other dependencies used in this starter.',
+        hidden: ({parent}) => parent.studioVersion === 3,
```
Comment (RitaDias) [inline schemas/documents/contributions/starter.js:297]:
```suggestion
        hidden: ({parent}) => parent.studioVersion !== 3,
```

**Pull request merged**